C_sharp_exercises_2/C_sharp_exercises_2/Exercises1.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+//Exercise 1:
+//Write a method in C# to display the characters and frequency of character from giving string
+namespace C_sharp_exercises_2
+{
+    public class Exercises1
+    {
+        public void CountingFrequencyCharacters()
+        {
+            Console.Write("input text:\t");
+            string inputstring = Console.ReadLine();
+            Dictionary<string, int> fruitsDictonary = new Dictionary<string, int>();
+            for (int i = 0; i < inputstring.Length; i++)
+            {
+                if (!fruitsDictonary.ContainsKey($"{inputstring[i]}"))
+                {
+                    int thisChar = inputstring.Where(x => (x == inputstring[i])).Count();
+                    fruitsDictonary.Add($"{inputstring[i]}", thisChar);
```

Comment:
i used dictionary for checking I already have a this letter in it or no  , if I will be used list or array I must check with  matrix and it will working longer .